console/st/console-st.sh

```diff
@@ -130,6 +132,7 @@ EOF
 
 stop_server()
 {
+	pkill $(basename "$SERVER")
```

Comment:
This is duplicated with next line?

Reply:
It is observed that the below pkill command is not able to kill the server process.
Hence added this to retry to kill the process during cleanup

Reply:
pkill -KILL may be used to forcibly kill it.